apps/ejabberd/src/mod_mam_muc_odbc_async_pool_writer.erl

```diff
@@ -61,7 +63,7 @@ worker_prefix() ->
 %% or
 %% `worker_count(_) = 16, partition_count() = 16'.
 worker_count(_Host) ->
```

Comment:
A leading underscore means "unused variable" and disables the compiler warning about that. Please don't use variables which are marked as unused / ignored, instead remove the underscore.